From ff3ccd19943796e58e9ac16db114a99dcaa8b4d6 Mon Sep 17 00:00:00 2001
From: Jonh Wendell <jonh.wendell@redhat.com>
Date: Mon, 21 Mar 2022 16:03:58 -0400
Subject: CNI: Remove the annotation scheduler.alpha.kubernetes.io/critical-pod
 (#38048)

It was removed in Kubernetes 1.16. We are already using its replacement,
`priorityClassName`.
---
 manifests/charts/istio-cni/templates/daemonset.yaml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/manifests/charts/istio-cni/templates/daemonset.yaml b/manifests/charts/istio-cni/templates/daemonset.yaml
index b8c034850b..5fbc022193 100644
--- a/manifests/charts/istio-cni/templates/daemonset.yaml
+++ b/manifests/charts/istio-cni/templates/daemonset.yaml
@@ -26,11 +26,6 @@ spec:
         k8s-app: istio-cni-node
         sidecar.istio.io/inject: "false"
       annotations:
-        # This, along with the CriticalAddonsOnly toleration below,
-        # marks the pod as a critical add-on, ensuring it gets
-        # priority scheduling and that its resources are reserved
-        # if it ever gets evicted.
-        scheduler.alpha.kubernetes.io/critical-pod: ''
         sidecar.istio.io/inject: "false"
         # Add Prometheus Scrape annotations
         prometheus.io/scrape: 'true'
-- 
2.35.3

